cni/pkg/iptables/iptables.go

```diff
@@ -57,6 +57,9 @@ type Config struct {
 	TraceLogging  bool `json:"IPTABLES_TRACE_LOGGING"`
 	EnableIPv6    bool `json:"ENABLE_INBOUND_IPV6"`
 	RedirectDNS   bool `json:"REDIRECT_DNS"`
+	// If true, TPROXY will be used for redirection. Else, REDIRECT will be used.
+	// Currently, this is treated as a feature flag, but may be promoted to a permanent feature if there is a need.
+	RedirectTPROXY bool `json:"REDIRECT_TPROXY"`
```

Comment:
nit: this is named a bit confusingly since REDIRECT is the alternative to TPROXY. Not sure I have a better name though

Reply:
`ENABLE_TPROXY`, `USE_TPROXY`, `TPROXY_CAPTURE`, `TPROXY_MODE`

agree that `REDIRECT_TPROXY` seems like it might be confusing for folks

Reply:
Hah, it made more sense when it was RedirectTPROXY until the casing becomes REDIRECT_TPROXY. :-(